apps/emqx_bridge/test/emqx_bridge_v2_SUITE.erl

```diff
@@ -133,6 +133,7 @@ setup_mocks() ->
     ok.
 
 init_per_suite(Config) ->
+    snabbkaffe:fix_ct_logging(),
```

Comment:
:scissors: ?

Reply:
No, I wasted a lot of time not seeing that emqx_resource_manager crashed. It just showed up inside the HTML logs for the tests and not on the console. I thought this would fix that but I don't think it did. @ieQu1 is not `snabbkaffe:fix_ct_logging()` supposed to make sure that all logging goes to the console?

Reply:
I meant to say: using that during local development is fine, but we shouldn't commit it there as it'll affect all subsequent test cases/suites.

Reply:
Ok, thanks, in that case I will remove it.

Reply:
Removed: [357b664](https://github.com/emqx/emqx/pull/11857/commits/357b664c8da626564623ff0bcddf8b16343b3370)